Exchange-Guide.md

```diff
@@ -0,0 +1,190 @@
+# Exchange Guide (WIP)
+
+## Goal
+This guide describes how to setup `xud` with an existing exchange system and how to connect to the XUD Simulation Network or `xud-simnet`. `xud` is in alpha stage and the simnet aims to give exchange operators a 'feel' of how things will work. `xud` in it's current stage should never be connected to a production system nor be used on mainnet. You'll be using test coins at all times. This guide purpusely is kept relatively manual to make it understandable how `xud` works. In parallel we are working on improved automated setup using docker.
+
+Once the setup is completed, you will be able to query pending orders, place orders, and experiment with `xud`'s rich set of commands. Once a match is found in the network, your orders will be executed via a cross-chain atomic swap. `xud-simnet` currently supports atomic swaps between BTC and LTC. This guide will show how to to install GO, bitcoin and litecoin daemons, setup Lightning daemons (LND) for bitcoin (BTC) and litecoin (LTC), setup `xud` and basic instructions of how to integrate its API and finally connect to `xud-simnet`. This guide is written for Linux & MacOS operating systems and geared towards IT administrators of exchanges.
+
+## Describing the Setup
+To connect to `xud-simnet` you'll need to run several components which together provide support for exchanging order information and atomic swaps between BTC and LTC on the lightning network. You will setup the following components:
+- `BTCD` - full node, connected to the BTC chain
+- `LTCD` - full node, connected to the LTC chain
+- `LND-BTC` - lightning network daemon for the BTC network
+- `LND-LTC` - lightning network daemon for the LTC network
+- `XUD` - Exchange Union's decentralized exchange layer
+
+## Installation 
+### Prerequisites
```

Comment:
If we want to, we could simplify this further by including a `install-deps` script.

Reply:
Yeah, but I'd say it's fine for now. Our goal should be docker to get rid of this. Btw do you have experience with docker? @erkarl

Reply:
Fair enough. Yes, I do have experience with Docker.